### DISCUSSION TITLE
Running this script with Cron Jobs

### DISCUSSION THREAD

**Discussion by risfat** [aapatre/Automatic-Udemy-Course-Enroller-GET-PAID-UDEMY-COURSES-for-FREE#176] [Q&A]
Is this even possible to run the script with Cron ?? I have tried...but it gets an error...the web driver doesn't load...Any help would be appreciated.Thank u.😊

**Comment by fakeid30** [comment]
I am not very familiar with cron. 

I can say that we use this : 

 - cron: '30 18 * * *'

In our CI which runs it everyday on 12AM indian time.

**Reply by risfat** [reply]
It seems to be not working....only for this python script....if i add other jobs to cron...then it does the work....

I think 🤔 in cron...the script couldn't open browser...thats why...its not working.....

**Reply by fakeid30** [reply]
Could be.

**Reply by cullzie** [reply]
@risfat Have you tried to export Display? Similar to this: https://stackoverflow.com/a/24707563
